app/src/utils/__tests__/numberFormatUtils.test.ts

```diff
@@ -1,4 +1,8 @@
-import { formatFloat, formatInt } from "../numberFormatUtils";
+import { formatFloat, formatInt, formatLatencyMs } from "../numberFormatUtils";
+
+const MS_PER_SECOND = 1000;
+const MS_PER_MINUTE = 60 * MS_PER_SECOND;
+const MS_PER_HOUR = 60 * MS_PER_MINUTE;
```

Comment:
place in https://github.com/Arize-ai/phoenix/blob/main/app/src/constants/timeConstants.ts